analytics/javaspark/src/main/java/mil/nga/giat/geowave/analytic/javaspark/sparksql/SimpleFeatureMapper.java

```diff
@@ -0,0 +1,61 @@
+package mil.nga.giat.geowave.analytic.javaspark.sparksql;
+
+import java.io.Serializable;
+import java.sql.Timestamp;
+import java.util.Date;
+
+import org.apache.spark.api.java.function.Function;
+import org.apache.spark.sql.Row;
+import org.apache.spark.sql.RowFactory;
+import org.apache.spark.sql.types.DataTypes;
+import org.apache.spark.sql.types.StructField;
+import org.apache.spark.sql.types.StructType;
+import org.opengis.feature.simple.SimpleFeature;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.vividsolutions.jts.geom.Geometry;
+
+import mil.nga.giat.geowave.analytic.javaspark.sparksql.util.GeomWriter;
+
+public class SimpleFeatureMapper implements
+		Function<SimpleFeature, Row>
+{
+	private static Logger LOGGER = LoggerFactory.getLogger(SimpleFeatureDataFrame.class);
+
+	private final StructType schema;
+	private final GeomWriter geomWriter = new GeomWriter();
```

Comment:
again, this is where its required to be serializable (we can consider marking it transient)